code/game/objects/items.dm

```diff
@@ -105,6 +105,10 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 	var/datum/rpg_loot/rpg_loot = null
 
+	var/in_inventory = FALSE//is this item equipped into an inventory slot or hand of a mob?
+	var/force_string = null//string form of an item's force
+	var/last_force_string_check = null

```

Comment:
I have used 'real' programming languages. I have much more experience in them than I do in BYOND. I just like initializing things to null/I don't like initializing things into a blank string or a 0 for reasons. That being said, I'll change it.